generators/app/templates/src/app/screens/Dashboard/screens/Home/index.js

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
+import withProvider from '~components/ProviderWrapper';
 
 import logo from './assets/logo.svg';
 import styles from './styles.module.scss';
+import { useSelector, Context, useDispatch } from './context';
+import { reducer, INITIAL_STATE, actionCreators } from './reducer';
 
 function Home() {
+  // Example of how to use these custom hooks
+  const foo = useSelector(state => state.foo);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(actionCreators.setFoo('React'));
+  }, [dispatch]);
```

Comment:
I know that you are just using this to set an example. But i think it is rather confusing because here you just need a setState. You really don't want to set "React" in all the context. Maybe we can think of a better example and leave this with setState. For example we could dispatch a list of technologies and use them in a children component like a button and show a warning.

Reply:
I realize it's not the best example, I just wanted to be expeditive. I don't think this is very important though, given that this component will be removed 100% of the times.